src/applications/static-pages/simple-forms/form-upload/App.js

```diff
@@ -59,8 +61,12 @@ export const App = ({ formNumber, hasOnlineTool }) => {
 
 App.propTypes = {
   hasOnlineTool: PropTypes.bool.isRequired,
-  toggleLoginModal: PropTypes.func.isRequired,
   formNumber: PropTypes.string,
+  shouldShow: PropTypes.bool,
 };
 
-export default App;
+const mapStateToProps = state => ({
+  shouldShow: toggleValues(state)[FEATURE_FLAG_NAMES.formUploadFlow],
```

Comment:
a few comments to consider:
- can use `useSelector` instead of `connect` approach if you like, since we're already using `useSelector`
- do we want to show [WIP](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/simple-forms/20-10207/containers/App.jsx) component if no feature toggle?
- also in that link above, there is `state?.featureToggles?.loading` if we want to use that